tests/v2/validation/clustertemplates/provisioning/clustertemplate_export_test.go

```diff
@@ -0,0 +1,105 @@
+package clustertemplates
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/v2/actions/clusters"
+	"github.com/rancher/rancher/tests/v2/actions/provisioning"
+	"github.com/rancher/rancher/tests/v2/actions/provisioninginput"
+	templates "github.com/rancher/rancher/tests/v2/validation/clustertemplates"
+	"github.com/rancher/shepherd/clients/rancher"
+	mgmt "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/extensions/clusters/kubernetesversions"
+	"github.com/rancher/shepherd/pkg/config"
+	"github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type ClusterTemplateExportTestSuite struct {
+	suite.Suite
+	client             *rancher.Client
+	session            *session.Session
+	provisioningConfig *provisioninginput.Config
+	cluster            *mgmt.Cluster
+}
+
+func (ct *ClusterTemplateExportTestSuite) TearDownSuite() {
+	ct.session.Cleanup()
+}
+
+func (ct *ClusterTemplateExportTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	ct.session = testSession
+
+	ct.provisioningConfig = new(provisioninginput.Config)
+	config.LoadConfig(provisioninginput.ConfigurationFileKey, ct.provisioningConfig)
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(ct.T(), err)
+
+	ct.client = client
```

Comment:
we now use a standard user by default in all provisioning tests. can you update this to use a standard user, or  is admin required?

Reply:
these are to be run with admin. as mentioned previously have a separate task for rbac checks,  standard user cases will be covered in that.

Reply:
but, we run provisioning checks with standard user by default. Can't standard user be used here to follow the same cadence? I don't think you'll be provisioning as anyone in the rbac tests, so this wouldn't be covered in the separate ticket.

Reply:
what if the standard user is the one who creates the template though?

Reply:
By default standard users cannot create rke cluster templates unless they have the additional global permissions `Create new RKE cluster templates` assigned to them.

Reply:
I see. Since this folder is for provisioning, I discussed with Markus and we agree that giving a standard user permission to the feature (you can do this in the setup part of the test) and using the standard user instead of admin would be ideal. for non-provisioning tests, it is fine to use the admin user.

Reply:
I still feel strongly that these need to be ran as a standard user with permissions to create rke1Templates